library/Notifications/Hook/ObjectsRendererHook.php

```diff
@@ -277,13 +277,15 @@ final public static function renderObjectLink(Objects $object): ?ValidHtml
             try {
                 $sourceType = $hook->getSourceType();
                 if ($object->source->type === $sourceType) {
-                    return $hook->createObjectLink($object->id_tags)
-                        ->addAttributes([
+                    $objectLink = $hook->createObjectLink($object->id_tags);
+                    if ($objectLink !== null) {
+                        return $objectLink->addAttributes([
                             'class' => [
                                 'icinga-module',
                                 'module-' . ($sourceType === 'icinga2' ? 'icingadb' : $sourceType)
                             ]
                         ]);
+                    }
```

Comment:
The loop should end in this case, otherwise we allow multiple implementations for the same source type.

Reply:
There is a return statement in the if condition.

Reply:
That was previously the case. Now it's conditional.